.github/workflows/build-tests.yml

```
@@ -68,6 +68,68 @@ jobs:
       ###########################
       - uses: actions/upload-artifact@v2
         with:
-          name: Build (${{ matrix.unityVersion }})
+          name: Build Ubuntu (${{ matrix.unityVersion }})
+          path: build
+          retention-days: 14
+
+  buildForAllPlatformsWindows:
+    name: Build for ${{ matrix.targetPlatform }} on version ${{ matrix.unityVersion }}
+    runs-on: windows-2019
+    strategy:
+      fail-fast: false
+      matrix:
+        projectPath:
+          - test-project
+        unityVersion:
+          - 2020.3.24f1
+        targetPlatform:
+          - StandaloneWindows64 # Build a Windows 64-bit standalone.
```

Can we add `StandaloneWindows` here as well? Just as an additional sanity check, since I think 32-bit windows build should obviously be able to be created with a windows runner:
```suggestion
          - StandaloneWindows64 # Build a Windows 64-bit standalone.
          - StandaloneWindows # Build a Windows 32-bit standalone.
```